core/overlays/ocp4-stage/common/configmaps.yaml

```diff
@@ -37,6 +37,9 @@ metadata:
 data:
   solvers: |
     solver-rhel-8-py36
+    solver-rhel-8-py38
+    solver-fedora-32-py37
```

Comment:
all at once?!

Reply:
we can switch off the fedora ones ?

Reply:
ja please do so, and lets observe what the load and workflows do...